tests/Serializer/SerializerTest.php

```diff
@@ -96,6 +97,73 @@ public function testNull(bool $serializeAllObjects): void
         $this->assertNull($result);
     }
 
+    public function testRegisteredObjectSerializers(): void
+    {
+        $object = new class() {
```

Comment:
Since this class does not require any argument in the constructor we can leave out the brackets

Reply:
:-1: with this, it would be less readable, and also maybe against the current CS.

Reply:
I don't think that's even possible and is an actual syntax error, maybe commented on the wrong line?

Reply:
It's not a syntax error, it's perfectly a valid syntax to write `new class {}`. I wouldn't say that it's less readable, but if you both agree on this feel free leave it as is

Reply:
Ah figured it out, for the anonymous class, yep, removed them.

Reply:
The line for me was wrong, however it's against CS, so I cannot change it and keep the build green, so I kept it in the end. I agree it could be left out, but phpcs says no.